index.html

```diff
@@ -4152,24 +4152,21 @@ <h2>Elements</h2>
 </ol>
 
 <p>To <dfn>get a known element</dfn> given <var>reference</var>:
+
 <ol class="algorithm">
-  <li>Let <var>node</var> be <a>get a node</a> with <a>current session</a>,
-  <a>current browsing context</a>, and <var>reference</var>.
+  <li>If not <a>node reference is known</a> with <a>current session</a>,
+  <a>current browsing context</a>, and <var>reference</var> return
+  <a>error</a> with <a>error code</a> <a>no such element</a>.
 
-  <li>If <var>node</var> is null, or <var>node</var> does not
-  implement {{Element}} return <a>error</a> with <a>error
-  code</a> <a>no such element</a>.
+  <li>Let <var>node</var> be the result of <a>get a node</a> with
+  <a>current session</a>, <a>current browsing context</a>, and
+  <var>reference</var>.
 
-  <li>If <var>node</var> <a>is stale</a>:
-   <ol>
-     <li><p>If <a>node reference is known</a> with <a>current
-     session</a>, <a>current browsing context</a>,
-     and <var>reference</var>, return <a>error</a> with <a>error
-     code</a> <a>stale element reference</a>.
+  <li>If <var>node</var> is not null and <var>node</var> does not implement
+  {{Element}} return <a>error</a> with <a>error code</a> <a>no such element</a>.
 
-     <p>Otherwise return <a>error</a> with <a>error
-     code</a> <a>no such element</a>.
-   </ol>
+  <li>If <var>node</var> is null or <var>node</var> <a>is stale</a> return
```

Comment:
Since this is apparently confusing, add a comment:
```<!-- since we already checked for known references, null here means that the object was GC'd -->```

Reply:
Sure. Let me know if the updated content that I just pushed is what you expect.